go-controller/pkg/ovn/policy.go

```diff
@@ -108,13 +108,24 @@ func hashedPortGroup(s string) string {
 }
 
 func (oc *Controller) syncNetworkPolicies(networkPolicies []interface{}) {
+	err := wait.PollImmediate(500*time.Millisecond, 60*time.Second, func() (bool, error) {
+		if err := oc.syncNetworkPoliciesRetriable(networkPolicies); err != nil {
+			klog.Errorf("Failed (will retry) in syncing network policies: %v", err)
+			return false, nil
+		}
+		return true, nil
+	})
+	if err != nil {
+		klog.Fatalf("Error in syncing network policies: %v", err)
+	}
+}
+
+func (oc *Controller) syncNetworkPoliciesRetriable(networkPolicies []interface{}) error {
```

Comment:
a good opportunity to add docstring ?

Reply:
How about
```
// This function implements the main body of work of syncNetworkPolicies.
// Upon failure, it may be invoked multiple times in order to avoid a pod restart.
```